static/js/components/MemoryLimitSelector.tsx

```diff
@@ -97,14 +130,21 @@ const MemoryLimitSelector: FC<Props> = ({
               onChange={(e) =>
                 setMemoryLimit({
                   ...memoryLimit,
-                  unit: e.target.value as MEM_UNITS,
+                  unit: e.target.value as BYTES_UNITS,
                 })
               }
               value={memoryLimit.unit}
             />
           </Col>
         </Row>
       )}
+      {maxMemory && (
+        <p>
+          <span>
+            Total memory: <b>{humanFileSize(maxMemory, true)}</b>
```

Comment:
I don't know if this value should also be formatted according to the currently selected unit (only for the `fixed (bytes)` case), what do you think? @edlerd

Reply:
Ok, done, but please check once again because I'm not sure I'm doing it the best way @ [0c1ec36](https://github.com/canonical/lxd-ui/commit/0c1ec3671413bf9b746bfc20e1931c167231b1f8) 
If it looks good, you can merge. Otherwise, I'm happy to improve it. :)